app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split()
```

Comment:
You can unpack the list which we will get as the result of split() method, like this

`variable_a, variable_b, variable_c = "a b c".split()`